vdr/ambassador.go

```diff
@@ -335,3 +341,20 @@ func (n ambassador) findKeyByThumbprint(thumbPrint []byte, didDocumentAuthKeys [
 	}
 	return documentKey, nil
 }
+
+// discoverPeers tries to find Nuts Network services in the given DID Document to add them as Nuts Network peers.
+func (n ambassador) discoverPeers(document did.Document, _ types.DocumentMetadata) error {
+	for _, service := range document.Service {
+		if service.Type == "nuts-network-grpc" {
```

Comment:
constant

Reply:
our docs should reflect this feature and enable admins to change the endpoint. If part of future development, it must be stated in an issue.